docs/updating-syntax.md

```diff
@@ -487,8 +487,14 @@ The process `when` section is deprecated. Use conditional logic, such as an `if`
 
 The process `shell` section is deprecated. Use the `script` block instead. The VS Code extension provides syntax highlighting and error checking to help distinguish between Nextflow variables and Bash variables.
 
+(updating-config-syntax)=
+
 ### Configuration syntax
 
+:::{versionadded} 25.02.0-edge
+The strict config syntax can be enabled in Nextflow by setting `NXF_ENABLE_STRICT_SYNTAX=true`.
```

Comment:
```suggestion
Enable the strict config syntax by setting `NXF_ENABLE_STRICT_SYNTAX=true`.
```
I wonder if this should be worded actively as the rest of the section relies on this being set. Don't consider this a blocker if you disagree.

Reply:
Yeah, I prefer this wording because the user might follow these migration notes with the language server or with Nextflow 25.04, so it's not essential that they enable this variable

Reply:
Valid - all good 👍 Thanks!